user-guide/getting-started.md

```diff
@@ -62,7 +62,7 @@ The Edge Policy Console must authenticate your session using a Kubernetes Secret
 
 For more information, see [Edge Control](../../reference/edge-control).
 
-## Configure TLS for Automatic HTTPS
+## Configure TLS Termination and Automatic HTTPS
```

Comment:
What I want to call out is the fact that AES will now automatically configure TLS by default. 

This has sucked for a couple of our users since AES does this silently and since networking is imperfect, all the failures they get don't necessarily point to TLS being configured at Ambassador being the issue. 

This section is a good place to call this out. Something like this would be effective.

## Configure TLS Termination and Automatic HTTPS

The Ambassador Edge Stack enables TLS termination by default using a self-signed certificate. If you have the ability to update your DNS, Ambassador can automatically configure a valid TLS certificate for you, eliminating the TLS warning. 

If you do not have the ability to update your DNS, skip to the next section, [Create a Mapping](#create-a-mapping).

If you want the Ambassador Edge Stack to just serve cleartext, see the [Host](../../reference/host-crd).

Reply:
okay added a blurb